theano/tensor/tests/test_opt.py

```diff
@@ -6088,7 +6091,9 @@ def test_local_useless_reshape():
     f = theano.function([i], m, mode=mode)
     topo = f.maker.fgraph.toposort()
     assert not any(isinstance(n.op, tensor.basic.Reshape) for n in topo)
-
+    theano.printing.debugprint(f, print_type=True)
+    # Check stacktrace was copied over correctly after opt was applied
+    assert check_stack_trace(f, ops_to_check='all')
```

Comment:
OK.
Although `local_useless_reshape` itself is not supposed to insert new Ops for now, it may do so in the future, so it's good to have a test.